packages/paste-website/src/components/Blockquote.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+
+interface Props {
+  name: string;
+  source: string;
+  sourceUrl: string;
+}
+
+const Citation: React.FC<Props> = ({name, source, sourceUrl}) => {
+  return (
+    <Text as="footer" fontWeight="fontWeightSemibold" marginTop="space40">
+      &mdash; {name}
+      {source && ', '}
+      {source && (
+        <cite>{sourceUrl ? <Anchor href={sourceUrl}>{source}</Anchor> : <Text as="span">{source}</Text>}</cite>
```

Comment:
Currently the typing of the `source` and `sourceUrl` props means that you cannot pass them as `null` or `undefined`.  Typescript will make sure they come through.  Therefore, these checks against `{source && ...}` will always be true.  In order to correct this, we first need to understand what we expect.  Do we expect the consumers of this component to use this a `source` and `sourceUrl`? Will this component be "correct" if they do? 

If we must always pass them, then simply remove the conditional check `source &&`.
If we don't need to always pass them, we can replace the typing as:
```
source?: string;
sourceUrl?: string;
```
and keep the checks.